airflow/api_connexion/endpoints/connection_endpoint.py

```diff
@@ -39,10 +39,12 @@
 from airflow.security import permissions
 from airflow.utils.session import NEW_SESSION, provide_session
 from airflow.utils.strings import get_random_string
+from airflow.www.decorators import action_logging
 
 
 @security.requires_access([(permissions.ACTION_CAN_DELETE, permissions.RESOURCE_CONNECTION)])
 @provide_session
+@action_logging(event=f"connection.{permissions.ACTION_CAN_DELETE.replace('can_','')}")
```

Comment:
Instead of doing this all over the place, we could change `action_logging` to something like this:

```python
def action_logging(*, event: str = "", permission: str = "") -> Callable[[T], T]:
    if permission.startswith("can_"):
        permission = permission[4:]
    if permission:
        event = f"{event}.{permission}"
    ...
```

And use it like this

```python
@action_logging(event="connection", permission=permissions.ACTION_CAN_EDIT)
```

Does this sound like a good idea?

----

An alternative approach would be to introduce a helper for this:

```python
def action_event_from_permission(prefix: str, permission: str) -> str:
    if permission.startswith("can_"):
        permission = permission[4:]
    if prefix:
        return f"{prefix}.{permission}"
    return permission
```

and do

```python
@action_logging(
    event=action_event_from_permission(
        prefix="connection",
        permission=permissions.ACTION_CAN_DELETE,
    ),
)
```